.hmy/wallet.ini

```diff
@@ -21,11 +21,17 @@ rpc = s3.t.hmny.io:14555
 
 [local]
 bootnode = /ip4/127.0.0.1/tcp/19876/p2p/Qmc1V6W7BwX8Ugb42Ti8RnXF1rY5PF7nnZ6bKBryCgi6cv
-shards = 1
+shards = 2
 
 [local.shard0.rpc]
 rpc = 127.0.0.1:14555
+rpc = 127.0.0.1:14557
```

Comment:
lgtm but can you explain me this field? does it represent the bootnode?

Reply:
those are rpc end points for wallet to connect.